elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/metadata/ColumnVisitor.java

```diff
@@ -79,7 +79,10 @@ protected T visitPhysicalReference(String reference) {
         List<String> references = new ArrayList<>();
 
         while (matcher.find()) {
-            references.add(matcher.group(1));
+            String value = matcher.group(1);
+            if (!value.startsWith("$$") && !value.startsWith("sql ")) {
+                references.add(value);
+            }
```

Comment:
Added this temporarily until ColumnVisitor is removed for test cases to work for SQL helper.